plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -262,6 +290,56 @@ def dcnm_get_ip_addr_info(module, sw_elem, ip_sn, hn_sn):
 #             raise Exception(json.dumps(msg_dict))
 
 
+# This call is used to get the details of the given fabric from the DCNM
+def get_fabric_details(module, fabric):
+    """
+    Used to get the details of the given fabric from the DCNM
+
+    Parameters:
```

Comment:
Looks like you are missing `module` as one of the parameters in this doc string

Reply:
Added